tests/acceptance/user-can-fill-out-hearing-form-test.js

```diff
@@ -335,4 +335,37 @@ module('Acceptance | user can save hearing form', function(hooks) {
 
     assert.equal(currentURL(), '/my-projects/4/hearing/done');
   });
+
+  test('if there is a server error when running .save(), user will see error message', async function(assert) {
+    this.server.create('disposition');
+    this.server.patch('/dispositions/:id', { errors: ['server problem'] }, 500); // force mirage to error
```

Comment:
This is super neat! Strangely I can't seem to find docs that talk about mocking these endpoints (`.get`, `.patch`) via `this.server`. Were you able to find any?

I did find this article, though... https://embermap.com/notes/86-testing-mirage-errors-in-your-routes
Also guessing it may have to do with Pretender: 
https://www.ember-cli-mirage.com/docs/api/modules/server~Server#pretender

Reply:
@godfreyyeung I mainly referenced this article: https://embermap.com/notes/86-testing-mirage-errors-in-your-routes

Reply:
yup, `server` is a reference to the Pretender instance, which is what powers Ember Mirage under the hood.

We have a sub to Ember Map if you need the p/w login, just use the labs_dl email (there's no password).